app/helpers/misc_helper.rb

```diff
@@ -0,0 +1,6 @@
+module MiscHelper
+  def get_age(dob)
+    now = Time.now.utc.to_date
```

Comment:
I think we should keep this just in the student model for now. If we ever end up needing to calculate age of something else we can move it somewhere else. For now, we are only dealing with students so move this implementation into `Student.age` method